src/.eslintrc

```diff
@@ -3,7 +3,11 @@
     "ecmaVersion": 6,
     "sourceType": "module"
   },
+  "plugins": [
+    "lit"
+  ],
   "extends": [
```

Comment:
With the `"plugin:lit/recommended"` you don't need to add the plugin:

https://github.com/43081j/eslint-plugin-lit/blob/master/src/configs/recommended.ts#L2

Reply:
Oh thanks @abdonrd! Updated.